src/client/src/MessageList.jsx

```diff
@@ -0,0 +1,20 @@
+import './styles/style.css'; 
+import moment from 'moment';
+
+export const MessageList = ({ messages }) => {
+  return (
+    <ul>
+      {messages.map(message => {
+        const dateString = message.createdAt;
+        const time = moment(dateString).format('HH:mm:ss');
+        return (
+          <li key={message.id}>
+            <span className="username">{message.userName}</span>
```

Comment:
Use destructuring assignment for message properties to improve code readability.